server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadEndpoint.feature

```diff
@@ -17,35 +17,58 @@ Feature: Download endpoint
     When "usera@domain.tld" checks for the availability of the attachment endpoint
     Then the user should be authorized
 
+  Scenario: An unauthenticated user should not have access to the download endpoint
+    When "usera@domain.tld" downloads "a1"
+    Then the user should not be authorized
+
+  Scenario: An authenticated user should not have access to the download endpoint without the authentication token
+    Given "usera@domain.tld" is connected
+    When "usera@domain.tld" downloads "a1" without the authentication token
+    Then the user should not be authorized
```

Comment:
`authenticated` means that the user has already access to the POST method, then he is trying to access the GET method.